eigenda/commitment.go

```diff
@@ -33,7 +33,7 @@ const (
 type Commitment []byte
 
 func (c Commitment) Encode() []byte {
-	return append([]byte{byte(op_plasma.GenericCommitmentType), byte(EigenDA), byte(EigenV0)}, c...)
+	return append([]byte{byte(EigenDA), byte(EigenV0)}, c...)
```

Comment:
We should be encoding the commitment server side

Reply:
I don't think this is the case. Let's hop on a call.